goal_src/jak1/pc/progress-pc.gc

```diff
@@ -1182,6 +1208,209 @@
+                      (send-event *target* 'get-pickup (pickup-type fuel-cell) (the float (game-task snow-eggtop)))
+                      ;; extra setup for IL speedruns
+                      (speedrun-set-common-settings)
+                      ))
+                    )
+                  (initialize! *game-info* 'game (the-as game-save #f) "ogre-start")
+                  )
+                 (((game-text-id village3-level-name))
+                  (when (-> *pc-settings* speedrunner-mode?)
+                    (set! *speedrun-setup-function* (lambda ()
+                      ;; give enough orbs to buy all cells 120+120+90+90+90+90=600
+                      (set! (-> *game-info* money-total) 600.0)
+                      (set! (-> *game-info* money) 600.0)
+                      ;; extra setup for IL speedruns
+                      (speedrun-set-common-settings)
+                      ))
+                    )
+                  (initialize! *game-info* 'game (the-as game-save #f) "village3-start")
+                  )
+                 (((game-text-id snow-level-name))
+                  (when (-> *pc-settings* speedrunner-mode?)
+                    (set! *speedrun-setup-function* (lambda ()
+                      ;; unlock flutflut
+                      (close-specific-task! (game-task beach-flutflut) (task-status need-resolution))
+                      (send-event *target* 'get-pickup (pickup-type fuel-cell) (the float (game-task beach-flutflut)))
+                      ;; unlock gondola
+                      (close-specific-task! (game-task village3-button) (task-status need-introduction))
+                      ;; extra setup for IL speedruns
+                      (speedrun-set-common-settings)
+                      ))
+                    )
+                  (initialize! *game-info* 'game (the-as game-save #f) "snow-start")
+                  )
+                 (((game-text-id cave-level-name))
+                  (when (-> *pc-settings* speedrunner-mode?)
+                    (set! *speedrun-setup-function* (lambda ()
+                      ;; unlock yellow eco vent
+                      (close-specific-task! (game-task snow-eggtop) (task-status need-resolution))
+                      (send-event *target* 'get-pickup (pickup-type fuel-cell) (the float (game-task snow-eggtop)))
+                      ;; extra setup for IL speedruns
+                      (speedrun-set-common-settings)
+                      ))
+                    )
+                  (initialize! *game-info* 'game (the-as game-save #f) "maincave-start")
+                  )
+                 (((game-text-id lavatube-level-name))
+                  (when (-> *pc-settings* speedrunner-mode?)
+                    (set! *speedrun-setup-function* (lambda ()
+                      ;; give 72 cells to unlock zoomer
+                      (set! (-> *game-info* fuel) 72.0)
```

Comment:
Im curious if these are even needed if we are hacking the task to complete? `(close-specific-task! (game-task lavatube-start) (task-status need-reward-speech))`

Reply:
Yeah we could remove those I think